packages/react/src/hooks/use-organizations.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from "@tanstack/react-query";
+import { useSlashID } from "./use-slash-id";
+
+export function useOrganizations() {
+  const { isAuthenticated, user } = useSlashID();
+
+  const result = useQuery({
+    queryKey: ["organizations", user?.oid],
+    queryFn: () => {
+      return user?.getOrganizations();
+    },
+    enabled: isAuthenticated,
```

Comment:
The query is in initial mode as long as this flag is false, it will automatically fetch data when `isAuthenticated` changes to `true`.